examples/python/PointCloud/PointCloudShowcase.py

```
@@ -0,0 +1,213 @@
+
+import time
+import os
+# os.environ["DEPTHAI_DEVICE_RVC4_FWP"] = "/home/tomas/code/depthai-device-kb/build_docker_arm64_rvc4/RelWithDebInfo/depthai-device-rvc4-fwp.tar.xz"
+import depthai as dai
+
+
+# ---------------------------------------------------------------------------
+# Print helpers
+# ---------------------------------------------------------------------------
+def print_header(title: str) -> None:
+    print(f"\n╔══════════════════════════════════════════════╗")
```

Ruff (0.15.5)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)